src/imitation/algorithms/adversarial/airl.py

```diff
@@ -92,7 +92,7 @@ def reward_train(self) -> reward_nets.RewardNet:
 
     @property
     def reward_test(self) -> reward_nets.RewardNet:
-        if isinstance(self._reward_net, reward_nets.ShapedRewardNet):
-            return self._reward_net.base
-        else:
-            return self._reward_net
+        reward_net = self._reward_net
+        while isinstance(reward_net, reward_nets.RewardNetWrapper):
```

Comment:
This is now sufficiently complex (& error-prone) I think it deserves a one-line comment explaining it

Reply:
`Recursively return the base network of the wrapped reward net`. Does this seem clear enough?

Reply:
It's clear what it does, but why do we do that? The point of reward_test is to get the unshaped version of the reward so I think there should be some mention of that.

Reply:
would be better to include it in the docstring